src/pages/Events/Events.scss

```diff
@@ -20,6 +20,10 @@
   }
 }
 
+.card-event {
+  min-width: 30%;
+}
+
 @media screen and (max-width: $breakpoint-tablet){
```

Comment:
This breakpoint is really large for the cards to start taking up the whole screen. We might consider maybe a mobile breakpoint instead? or have the cards go from 3 per row to 2 per row to screen wide?